src/scss/components/button/_mixins.scss

```diff
@@ -13,10 +13,14 @@
   padding: 0;
 }
 
-@mixin button-basic {
+@mixin button-base {
   border-radius: 2px;
   display: inline-flex;
   justify-content: center;
+}
+
+@mixin button-common { // less: #Button.basic()
```

Comment:
Renamed `button-basic` to `button-common` because `button-base` and `button-basic` are too similar.
Additionally, styles for `button-base` better reflect the name.